redhat_upgrade_tool/rollback/bootloader.py

```diff
@@ -27,10 +27,11 @@ def check_call(*popenargs, **kwargs):
     "config-{0}",
 ]
 
+_BOOM_UTIL_PATH = "/usr/libecex/boom"
```

Comment:
`libecex`

Reply:
I used old build for testing. Thanks for really good catch!!

Reply:
Frankly I liked `libecex`, reminds me of "libeček" (Levisticum officinale).

Reply:
:-D I read that for the first time as "libecek" :-D